### PR TITLE
fixes #10820 - collect tasks dump with foreman-debug

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: deploy/script/katello-debug.sh [hunk 1]
```diff
@@ -61,6 +61,13 @@ fi
 add_files /etc/gofer
 add_files /var/log/gofer
 
+#foreman-tasks export
+if hash foreman-rake  2>/dev/null; then
+  echo "Exporting tasks, this may take a few minutes."
+  tasks_filename=`foreman-rake foreman_tasks:export_tasks | tail -n 1 | awk '{print $2}'`
```
Comment (ehelms) [inline deploy/script/katello-debug.sh]: Is this not a default task? I tried testing this on my recent 2.2.1 and when I run this manually it tells me there is no task.

Comment (jlsherrill) [inline deploy/script/katello-debug.sh]: Its awaiting a new build of foreman-tasks:

https://github.com/theforeman/foreman-tasks/commit/3755354bea374cd230bd5b4a05e8c662ed4fb5de

Comment (ehelms) [inline deploy/script/katello-debug.sh]: Do you think this should do a check of the return of the command or that it exists? In the current state (of non existent rake task), it leads you to believe its exporting the tasks successfully in the output and then you end up with nothing without an obvious reason that it failed here.

Comment (jlsherrill) [inline deploy/script/katello-debug.sh]: updated to capture stderr from that command, so if it fails we'll know why

**Comment by ehelms** [comment]
APJ

**Pull request merged**